dali/pipeline/executor/source_info_propagation_test.cc

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/executor/source_info_propagation.h"
+#include <gtest/gtest.h>
+#include <cassert>
+#include <memory>
+#include <string>
+#include <vector>
+#include "dali/core/format.h"
+#include "dali/core/span.h"
+#include "dali/pipeline/workspace/workspace.h"
```

Comment:
Shouldn't all the includes from dali be at the bottom according to GSG/linter?

Reply:
Yeah, that one kept me baffled, too, but apparently, the linter knows what to do with the include that matches the `.cc` file and can somehow skip `_test`.